templates/shift-scheduling/output.json

```diff
@@ -0,0 +1,35 @@
+{
+  "version": { "sdk": "v0.28.0" },
+  "solutions": [
+    {
+      "status": "optimal",
```

Comment:
I don't see how this solution satisfies the constraints. The shifts are not being covered with the required number of workers...

Reply:
I see, so the demand constraint has slack variables. It would be helpful to print out the value of these slack variables for each shift that is missing. Basically, would be useful to know, in the same manner as we have the unplanned stops, how many workers are missing for shifts that are undersupplied (or viceversa).